grpc/java-diatheke/liblite/build.gradle

```diff
@@ -0,0 +1,73 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish' // Lets us push packages up to GitHub Packages
+    id 'net.linguica.maven-settings' version '0.5' // lets us pull maven settings from ~/.m2/
+    id 'com.google.protobuf'
+}
+
+group = 'com.cobaltspeech.diatheke'
+version = '2.0.0'
```

Comment:
Is this a string that will need to be updated every time we tag sdk-diatheke?

Reply:
Yes.  Changing that should be made part of the top level readme file.  Although, I think that should also be migrated to the makefile.  Something like `make bumptheversion VERSION=2.0.0`, and have that change the files for each language in a more automated way.
I figured I would do that in a separate PR.  That pattern will likely apply to more than just this sdk, and there were some changes to the hugo docs stuff that also affected the version changing steps.